temporalio/converter.py

```diff
@@ -133,7 +154,7 @@ def __init__(self, *converters: PayloadConverter) -> None:
         # Insertion order preserved here
```

Comment:
(Python >=3.7 feature)

Reply:
Which is our minimum version

Reply:
Yup, just noting that, could add to the comment.

Reply:
Will do

Reply:
Added to comment